lib/class-wp-rest-menus-controller.php

```diff
@@ -531,6 +533,26 @@ function handle_auto_add( $menu_id, $request ) {
 		return $update;
 	}
 
+	/**
+	 * Returns names of the locations assigned to the menu.
+	 *
+	 * @param int $menu_id The menu id.
```

Comment:
Add a since and a unit test.

Reply:
Won't be `since` tags updated if/when this endpoint gets merged into WP core?

I will work on unit tests.

Reply:
Added unit test.

Reply:
Added since.